dtmi/azure/iot/deviceupdate-1.json

```diff
@@ -0,0 +1,163 @@
+{
+  "@id": "dtmi:azure:iot:deviceUpdate;1",
+  "@type": "Interface",
+  "description": "Device Update Interface",
```

Comment:
Should the description format follow the guidelines? https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines#interfaces

Reply:
Fixed description